cmd/ops_agent_windows/main_windows.go

```diff
@@ -57,6 +58,16 @@ func main() {
 				log.Fatal(err)
 			}
 			infoLog.Printf("uninstalled services")
+		} else if *healthChecks {
+			healthCheckResults := getHealthCheckResults()
+			for _, result := range healthCheckResults {
+				if result.Err != nil {
+					infoLog.Printf("Error: %s", result.Message)
```

Comment:
Does `result.Err` contain anything relevant that we might want to include in the printed text?

Reply:
Discussed offline; `result.Message` already embeds the contents of `result.Err`.

I would suggest a small refactor of `HealthCheckResult`:
* Get rid of the `Message` field and change it into a `String()` function that generates the formatted message on-demand. This makes it so that we don't have to manually keep `Err` and `Message` in sync, with the added benefit of implementing `Stringer`.
* Add a `Name` field to support the above function since it's needed for generating the message.

WDYT?

Reply:
I really like this idea of generating the formatted message on demand. I was trying to figure out where to put this and `result` seems to be great (it has all the info, with the `Err` and name it should be enough).

Reply:
Spoke offline, will be part of https://github.com/GoogleCloudPlatform/ops-agent/pull/1139